data/pc/common/protocolVersions.json

```diff
@@ -1,4 +1,10 @@
 [
+  {
+    "minecraftVersion": "1.13.1",
+    "version": 401,
+    "usesNetty": true,
+    "majorVersion": "1.13.1"
+  },
```

Comment:
this file must be exhaustive, please complete with https://wiki.vg/Protocol_version_numbers